src/components/Person/Person.jsx

```diff
@@ -1 +1,68 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const misha = {
```

Comment:
so, you move this to the app & after that put this object on the props on the Person component
So, you have 3 components on the top
It means, that you create these objects 3 times)

Reply:
Move to the separate file on array & use map for this in App component

Reply:
I'm not sure that I understood correctly, but I made a separate file json for data and used map in App component